sql/mob_skill_lists.sql

```diff
@@ -4155,7 +4155,15 @@ INSERT INTO `mob_skill_lists` VALUES ('Exoplate_Phase_3',2059,998);
 INSERT INTO `mob_skill_lists` VALUES ('Exoplate_Phase_3',2059,999);
 INSERT INTO `mob_skill_lists` VALUES ('Exoplate_Phase_3',2059,1000);
 
--- Next ID : 2060
+INSERT INTO `mob_skill_lists` VALUES ('Beelzebub',2060,334);
+INSERT INTO `mob_skill_lists` VALUES ('Beelzebub',2060,335);
+
+INSERT INTO `mob_skill_lists` VALUES ('Hell_Fly',2061,334);
+INSERT INTO `mob_skill_lists` VALUES ('Hell_Fly',2061,335);
+
```

Comment:
Are we totally sure these mobs don't use final sting? The capture sadly didn't experiment much with mob skills or resistances. We also don't need to make a skill list for each mob - if it is the case they don't use final sting, should only need one skill list that they both share.